pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -190,7 +190,7 @@ func BuildDefaultAPIServer(options configapi.MasterConfig) (*apiserveroptions.Se
 		master.DefaultAPIResourceConfigSource(),
 	)*/
 	// the order here is important, it defines which version will be used for storage
-	storageFactory.AddCohabitatingResources(batch.Resource("jobs"), extensions.Resource("jobs"))
+	storageFactory.AddCohabitatingResources(extensions.Resource("jobs"), batch.Resource("jobs"))
```

Comment:
Jeezas, no 😨 ! Thanks for pointing it out. Must have slipped when testing that job storage migration story.